backend/src/test/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhirTest.java

```diff
@@ -538,13 +539,14 @@ void convertExistingCsv_populatesBlankFields() {
 
   @Test
   void convertExistingCsv_setsOrderingProviderTimezone_forNoTimezoneInDates() {
-    when(resultsUploaderCachingService.getZoneIdByAddress(any()))
-        .thenReturn(ZoneId.of("US/Mountain"));
+    ZoneId zoneId = ZoneId.of("US/Mountain");
+    when(resultsUploaderCachingService.getZoneIdByAddress(any())).thenReturn(zoneId);
     InputStream input = loadCsv("testResultUpload/test-results-upload-valid-dates-no-timezone.csv");
 
-    Instant orderTestDate = Instant.parse("2021-12-19T12:00:00-07:00");
```

Comment:
⚠️ Went back and changed this in this file, @mehansen, based on your previous comment about these tests failing when it is DST 😅 Thank you for catching that!